docs/swagger/spec.yml

```diff
@@ -2331,7 +2446,71 @@ definitions:
       maxModifiedDatetime:
         type: string
         format: datetime
+  UploadCreate:
+    type: object
+    properties:
+      uploadStatus:
+        type: string
+        description: Status of upload
+        enum:
+          - CREATED
+          - UPLOADING
+          - PROCESSING
```

Comment:
Should there be a `QUEUED` status between `UPLOADING` and `PROCESSING`, or is it not necessary to expose that?